File: Sem4/MPP/java/lab11RESTjava/Service/src/main/java/ubb/scs/mpp/Service/IAppService.java
package ubb.scs.mpp.Service;

import ubb.scs.mpp.Service.observer.Observer;
import ubb.scs.mpp.model.Angajat;
import ubb.scs.mpp.model.Spectacol;

import java.util.Date;

public interface IAppService {

    public Angajat login(Angajat angajat, Observer observer) throws ServiceException;
    public Iterable<Spectacol> allSpectacol() throws ServiceException;
    public Iterable<Spectacol> getFilteredSpectacol(Date day) throws ServiceException ;
    public void addBilet(String buyer,Spectacol spec,int seats) throws ServiceException;
    public void logout(Angajat angajat) throws ServiceException;



}

File: Sem4/MPP/cs/lab9server/Services/IAppService.cs
using Domain;
using WinFormsApp1.Service;


public interface IAppService
{
    public Angajat Login(Angajat angajat,Observer observer);
    public IEnumerable<Spectacol> GetSpectacols();
    public IEnumerable<Spectacol> GetSpectacolsByDate(DateTime date);
    public void AddBilet(long spectacolId, int seats, String name);
    public void logout(Angajat angajat);

}